src/cli/cli.cpp

```diff
@@ -2621,14 +2639,33 @@ otError Interpreter::ProcessMasterKey(uint8_t aArgsLength, Arg aArgs[])
 
     if (aArgsLength == 0)
     {
-        OutputBytes(otThreadGetMasterKey(mInstance)->m8);
+        const otMasterKey *masterKey = otThreadGetMasterKey(mInstance);
+
+        if(otPlatCryptoGetType() == OT_CRYPTO_TYPE_PSA)
+        {
+            uint8_t mMasterKeyBuffer[OT_MASTER_KEY_SIZE];
```

Comment:
Why are those variables prefixed with `m` while they're not struct members?

Reply:
Ahh looks like a typo. I will correct these.